apps/homescreen/test/unit/pages_test.js

```diff
@@ -314,6 +321,34 @@ suite('Pages', () => {
       };
       pages.unpinSelectedCard();
     });
+
+    suite('in case of DataStore conflict', function(done) {
```

Comment:
Either we need a comment to explain this or this should match the rest of the file and use fat-arrow functions (I prefer the latter, but if things will break, obviously the former :)) - otherwise I could see myself or someone else changing this to be consistent with the rest of the file. Seems like we're working around a bug elsewhere by using `function`.